winpr/libwinpr/utils/asn1/asn1.c

```diff
@@ -1023,22 +1023,33 @@ static size_t WinPrAsn1DecReadIntegerLike(WinPrAsn1Decoder* dec, WinPrAsn1_tag e
 	if (len == 0 || Stream_GetRemainingLength(&dec->source) < len || (len > 4))
 		return 0;
 
-	WinPrAsn1_INTEGER val = 0;
+	union
```

Comment:
What a price to not have compiler warnings ! There's no warning, but also no one who understand with a quick look what the code is doing...

Reply:
true, not required here.
but reading a `union` is not really hard, don´t exaggerate ;)